src/global-styles.js

```diff
@@ -0,0 +1,97 @@
+import { createGlobalStyle } from "styled-components";
```

Comment:
The name that I usually use for the main css style is either `main` or `app`, but if this is some sort of standard connected with `styled components`, then let's use it. I think Style in the name is needed as later this is used as component in `index.tsx`. It would look weird to import App component 😁
I am wondering where to keep this file, I think it could fit better in `styles` folder (unless styled components standard as well).

In general I was taking a look and I think I'd keep a bit different structure, something along those lines:
```
 assets > styles > breakpoints.js & colors.js
 assets > img > icons > all icons here
 assets > img > all background images here
 assets > fonts > all fonts here
```
WDYT? :)

Reply:
Ad 1 I would rather stay with name global-styles - it's not connected with styled-components ;) It says more to me - especially that it's js file. 

Ad 2 I like the new structure - good idea.